Makefile

```diff
@@ -44,19 +45,12 @@ help:
 	@echo " * 'uninstall'      - Remove installed binaries from system locations"
 	@echo " * 'version'        - Print current cri-containerd release version"
 
-.PHONY: check-gopath
-
-check-gopath:
-ifndef GOPATH
-	$(error GOPATH is not set)
-endif
-
 verify: lint gofmt boiler
 
 version:
 	@echo $(VERSION)
 
-lint: check-gopath
```

Comment:
was only necessary to do this test prior to go 1.8 and we require 1.8 so.. yeah ok to remove :-) Maybe to a go version check?